src/main/proto/Report.proto

```diff
@@ -127,4 +127,7 @@ message JiraTickets {
     string priority = 6;
     string reporter = 7;
     string summary = 8;
+    string viewTicket = 9;
```

Comment:
should be snake case based on the convention https://developers.google.com/protocol-buffers/docs/style#message-and-field-names